config/backup-manager.php

```diff
@@ -15,10 +15,11 @@
     ],
     'gcs' => [
         'type' => 'Gcs',
-        'key'    => '',
-        'secret' => '',
-        'bucket' => '',
-        'root'   => '',
+        'project' => env('GOOGLE_CLOUD_PROJECT_ID', 'your-project-id'),
```

Comment:
using the same name of environment variables. https://github.com/Superbalist/laravel-google-cloud-storage